community/saas-companies-engagement-procedure.md

```diff
@@ -0,0 +1,118 @@
+community.
+
+The goal is to provide a clear and consistent process for engaging with 
+SaaS companies that are included but not limited to those that use SPIFFE and 
+SPIRE, provide a proprietary SPIFFE-compatible service, or are interested in 
+contributing to the SPIFFE project.
+
+More specifically, this document includes guidelines for communication, 
+technical contact designation, and handling potential disruptive participants.
+
+### Technical Contacts
+
+#### Objective
+
+To ensure effective communication and support for SaaS consumers 
+using the SPIFFE specification by designating responsible technical contacts.
+
+#### Guidelines
+
+##### Designation of Contacts
+
+Each SaaS company that directs customers to the SPIFFE Slack channel must 
+designate two technical contacts.
+
+These contacts will serve as the primary point of contact for addressing queries 
+and facilitating communication between the customers and the Spiffy community.
+
+##### Documentation of Contacts
+
+A dedicated folder for each company will be created in the SPIFFE repository
+under the `./community/saas-technical-contacts` directory.
+
+The folder will contain:
+
+* Names and contact information of the designated technical contacts.
+* Guidelines for interaction and support procedures.
+* Any relevant documentation provided by the company.
+
+### Interaction Protocols
+
+#### Objective
+
+To establish clear protocols for interactions between SaaS consumers and the 
+SPIFFE community to ensure a productive and respectful environment.
+
+#### Guidelines
+
+##### Communication Etiquette
+
+All interactions should be professional, respectful, and focused on technical 
+issues.
+
+Technical contacts are responsible for ensuring that their customers, and anyone
+who join the community through their reference adhere to the community 
+guidelines and maintain a positive environment.
+
+##### Issue Resolution
+
+Technical contacts should facilitate the resolution of technical issues by 
+providing detailed information and context.
+
+For complex issues, technical contacts should coordinate with SPIFFE maintainers 
+and contributors to find solutions.
+
+##### Feedback Mechanism
+
+SPIFFE Steering Committee encourages SaaS consumers to provide feedback on their 
+experience with the SPIFFE specification and the support received.
+
+SPIFFE Steering Committee will use this feedback to improve documentation, 
+support processes, and community guidelines.
+
+### Managing Disruptive Participants
+
+#### Objective
+
+To identify and manage participants who disrupt the community, ensuring that 
+the SPIFFE Slack channel remains a productive and positive space.
+
+#### Guidelines
+
+##### Identification of Disruptive Behavior
+
+Disruptive behavior includes, but is not limited to, spamming, harassment, and 
+consistently off-topic discussions.
+
+Technical contacts should monitor interactions and identify any disruptive 
```

Comment:
If we identify that one of our customers is responsible for disruptive behaviour, what actions should we take? Should we determine that this behaviour breaches the conduct guidelines ourself and reach out directly to the customer, or should we we to escalate this to the SPIFFE team first?

In the case that we don't identify this behaviour before SSC do, should we expect to be contacted to deliver the warning?

Reply:
Here's what I have in mind:

* If a technical contact identifies disruptive behavior, they should first determine * if this behavior breaches the conduct guidelines.
* Upon confirming a breach, the technical contact should reach out directly to the customer to deliver a warning and outline the expected conduct.
* If the behavior persists after the warning, they should escalate the issue to the SPIFFE Steering Committee (SSC).



I’ll amend the document too.